plugins/auth-backend/src/identity/DatabaseKeyStore.ts

```diff
@@ -66,7 +66,13 @@ export class DatabaseKeyStore implements KeyStore {
     return {
       items: rows.map(row => ({
         key: JSON.parse(row.key),
-        createdAt: utc(row.created_at),
+        createdAt: DateTime.fromFormat(
+          (row.created_at as unknown) as string,
+          'yyyy-MM-dd HH:mm:ss',
```

Comment:
is it really stored on this form? i would have hoped for full `yyyy-MM-ddYHH:mm:ss.SSSZ` with fractional seconds and timezone ...

Reply:
oh i guess this is what knex deserializes timestamps as

Reply:
You can do Luxon.fromSql if I remember correctly.

Reply:
Sorry `DateTime.fromSql`

Reply:
https://moment.github.io/luxon/docs/manual/parsing.html#sql

Reply:
I thought that it would be a normal JS date object so I first tried to use `DateTime. fromJSDate ` but then I got an invalid DateTime object.

`typeOf creted_at` -> `string`

Reply:
No knex will just return string. If you want to get a `DateTime` from that use `DateTime.fromSQL` and give it `created_at` :)

Reply:
The string did not contain information about timezone or fractional seconds.

Reply:
Is it because the `created_at` in the schema is only `Date` and not  `DateTime`?

Reply:
Looks llike it:

```ts
table
      .timestamp('created_at', { useTz: false, precision: 0 })
      .notNullable()
```

Looks like it's only day information stored, whereas someplace else is `DateTime`

```ts
    table
      .dateTime('created_at')
      .defaultTo(knex.fn.now())
      .notNullable()
      .comment('The timestamp when this task was created');
```

Reply:
@freben @Rugvip should we alter the auth-table to be `DateTime` or is it fine being days?

Reply:
@benjdlambert `fromSQL` worked, I have updated the PR.

Reply:
@​benjaminr seems like the timestamp is precise enough in postgres at least, seeing `2021-02-04 16:37:49.51418`